OWNERS.md

```diff
@@ -0,0 +1,8 @@
+# Owners for ratify-project
+
+Owners:
```

Comment:
consider removing this at first and ask self-nomination/ nomination issues be made prior

Reply:
If I understand correctly, this file lists Ratify project owners/maintainers. How about owners/maintainers for this `.github` repo, it could be different. (It seems Ratify project use the term owner, not maintainer based on the governance document)

Reply:
We can take `MAINTAINERS`, `OWNERS` and `CODEOWNERS` in `https://github.com/oras-project/oras` as examples.

Reply:
thanks for suggestion! I'll remove owners/maintainers docs in this PR as we have not nominated them yet. Once we have nomination completed, I'll add those files.